code/scripting/scripting.h

```diff
@@ -139,26 +101,17 @@ struct script_condition
 	// CHC_SHIPCLASS, CHC_WEAPONCLASS - stores the index of the info object requested by the condition
 	// CHC_VERSION, CHC_APPLICATION - stores validity of the check in 1 for true or 0 for false, as the condition will not change after load.
 	// see ConditionedHook::AddCondition for exact implimentation
-	int condition_cached_value = -1;
+	int condition_cached_value;
```

Comment:
Because I want the ``script_condition`` struct to be list-constructible as done in ParseCondition later. This is only possible if all members don't have defaults, at least on some GCC's, IIRC.
Plus, since all ``script_condition`` objects are initialized like this, with this int being directly supplied by the result of ``cache_condition``, it should be fine.

Reply:
gotcha